src/azure-cli/HISTORY.rst

```diff
@@ -16,6 +16,10 @@ Release History
 * Add optional parameter to specify replica SKU for create replica command.
 * Fix the issue with CI test failure with creating MySQL replica.
 
+**Backup**
+
+*Updated recovery services SDK to 0.4.0
```

Comment:
If there are no functional changes, this can be omitted entirely. If there are, this section should fall in alphabetical order with the other sections.

Reply:
Removed